### DISCUSSION TITLE
Rust client

### DISCUSSION THREAD

**Discussion by balegas** [electric-sql/electric#566] [Feature requests]
We're already working on a Rust client. 
Current state is that we have a driver for sqlx and are looking into moving part of Electric logic that now lives in the TS land into WASM, so that we (and the community) can build clients for other languages without rewriting the more complex logic.

**Comment by J05HI** [comment]
@balegas so it should work with Tauri when ready?

**Reply by balegas** [reply]
We have a working demo of electric w/ Tauri. We need to do more work to be able to share it.

Let me know if you’re interested in working on it.

**Comment by trydalch** [comment]
Did the Electric logic alluded to above make it's way into WASM? 

I'd be interested in developing a c# client. Would be even more interested if there was a brief .md on best practices for new clients, etc.

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
